src/btree/row_srch.c

```diff
@@ -785,7 +785,7 @@ __wt_row_random_leaf(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt)
  *	Find a random leaf page in a row-store tree.
  */
 int
-__wt_row_random_descent(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt)
```

Comment:
I'm not seeing much here that is row-store specific.  If we change the `page->type` check to use `WT_PAGE_IS_INTERNAL(page)`, can we use this on any tree?